src/Refinery/Integer/LessThanOrEqual.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+/* Copyright (c) 2021 Luka Stocker <luka.stocker@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Refinery\Integer;
+
+use ILIAS\Refinery\Constraint;
+use ILIAS\Data;
+use ILIAS\Refinery\Custom\Constraint as CustomConstraint;
+
+class LessThanOrEqual extends CustomConstraint implements Constraint
+{
+    /**
+     * @var int
+     */
+    protected $max;
```

Comment:
```suggestion
    protected int $max;
```
... and remove docstrings. Yay, PHP 7.4.